src/main/java/com/google/sps/storage/StorageWrapper.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.storage;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
+
+/**
+ * Implementation of the StorageWrapperInterface, with the purpose of providing I/O to
+ * the Google Cloud Datastore component.
+ */
+public class StorageWrapper implements StorageWrapperInterface<Gson, String> {
```

Comment:
Coupling concern - Using Gson as the EntryType will have implications on the interface for our REST API. I think it will be safer to use **String** type here and use GSON on our RestAPI to serialize the information.